com.unity.render-pipelines.universal/Runtime/UniversalRendererData.cs

```diff
@@ -224,19 +221,6 @@ internal TileSize tileSize
             }
         }
 
-        /// <summary>
-        /// Controls when URP renders via an intermediate texture.
-        /// </summary>
-        public IntermediateTextureMode intermediateTextureMode
```

Comment:
We can't remove public API on this backport. Let's make it obsolete.

Reply:
Done